test/e2e/page-objects/pages/dialog/select-network.ts

```diff
@@ -187,16 +191,12 @@ class SelectNetwork {
 
   async clickDiscoverButton(): Promise<void> {
     console.log('Click Discover button in network options');
-    await this.driver.clickElement(
-      '[data-testid="network-list-item-options-discover"]',
-    );
+    await this.driver.clickElement(this.discoverButton);
   }
 
   async check_discoverButtonIsVisible(): Promise<void> {
     console.log('Check Discover button is visible in network options');
-    await this.driver.waitForSelector(
-      '[data-testid="network-list-item-options-discover"]',
-    );
```

Comment:
in page object modal, we should define selectors as class variable to be reusable